app/views/collections/_collections.html.slim

```diff
@@ -13,6 +13,8 @@
   .card.my-4
     .card-header
       h6
+        - if current_user.favorite_collections.include?(collection)
+          i.fa-solid.fa-star>
```

Comment:
What is the trailing `>` good for?

Reply:
It adds a trailing whitespace. In this case:
without `>`
![image](https://github.com/openHPI/codeharbor/assets/24592181/ad343259-a742-4bd7-94d0-7e1d05cb139a)
with '>'
![image](https://github.com/openHPI/codeharbor/assets/24592181/d8081e92-82b1-485f-affb-11d449bdc5cd)

Reply:
I see. I know what it does for expressions, e.g. `=>` but have not seen it used for classes.